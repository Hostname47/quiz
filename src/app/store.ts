import {configureStore} from '@reduxjs/toolkit';
import userSlice from '../features/user/userSlice';
import gameSlice from '../features/game/gameSlice';
import appSlice from '../features/app/appSlice';

const store = configureStore({
  reducer: {
    app: appSlice,
    user: userSlice,
    game: gameSlice,
  },
});

export default store;

export type RootState = ReturnType<typeof store.getState>;
export type AppDispatch = typeof store.dispatch;
