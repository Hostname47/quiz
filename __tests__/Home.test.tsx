import {NavigationContainer} from '@react-navigation/native';
import {render} from '@testing-library/react-native';
import Home from '../src/screens/Home';
import {Provider} from 'react-redux';
import store from '../src/app/store';

const navigation = {navigate: jest.fn()};

describe('Testing navigation actions', () => {
  it('Should match snapshot', () => {
    const {toJSON} = render(
      <Provider store={store}>
        <NavigationContainer>
          <Home navigation={navigation} />
        </NavigationContainer>
      </Provider>,
    );

    expect(toJSON()).toMatchSnapshot();
  });
});
