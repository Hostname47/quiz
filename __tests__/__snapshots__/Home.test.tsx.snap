// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing navigation actions Should match snapshot 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "backgroundColor": "#373a42",
        "borderBottomColor": "#5a6070",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "height": 76,
      }
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#40444f",
          "justifyContent": "center",
          "opacity": 1,
          "paddingHorizontal": 12,
          "paddingVertical": 4,
          "width": 80,
        }
      }
    >
      <Image
        source={
          {
            "testUri": "../../../src/assets/avatars/6.png",
          }
        }
        style={
          {
            "backgroundColor": "#a0b2bc",
            "borderRadius": 30,
            "height": 40,
            "marginBottom": 4,
            "width": 40,
          }
        }
      />
      <Text
        numberOfLines={1}
        style={
          {
            "fontSize": 11,
          }
        }
      >
        - - -
      </Text>
    </View>
    <View
      style={
        {
          "backgroundColor": "#5a6070",
          "width": 1,
        }
      }
    />
    <View
      style={
        {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "center",
        }
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "backgroundColor": "#2b2d33",
            "borderRadius": 50,
            "flexDirection": "row",
            "opacity": 1,
            "padding": 6,
          }
        }
      >
        <RNSVGSvgView
          align="xMidYMid"
          bbHeight="24"
          bbWidth="24"
          fill="#6cdd6e"
          focusable={false}
          meetOrSlice={0}
          minX={0}
          minY={0}
          style={
            [
              {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              {
                "height": 24,
                "width": 24,
              },
              {
                "flex": 0,
                "height": 24,
                "width": 24,
              },
            ]
          }
          vbHeight={260}
          vbWidth={260}
        >
          <RNSVGGroup
            fill={
              {
                "payload": 4285324654,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          >
            <RNSVGPath
              d="M142.19 193.16c0 4.61.27 8.74-.05 12.83-.58 7.42-5.3 11.89-12.11 11.91s-11.58-4.45-12.16-11.84c-.32-4.08-.05-8.2-.05-13.34-6.41 0-12 .05-17.65 0-8.61-.1-13.74-4.55-13.88-11.93-.13-7.57 5.24-12.56 13.93-12.63 13.41-.1 26.82 0 40.23 0 9.14 0 14.68-4.68 14.91-12.39.24-8.07-5.18-13.1-14.6-13.32-8.67-.21-17.38.22-26-.42-20.18-1.54-34.76-17.3-34.86-37.14-.1-19.5 14.18-35.29 33.92-37.51 1-.11 1.92-.28 3.89-.56 0-3.91-.16-7.78 0-11.63.42-8.24 5-13.07 12.21-13.09s11.84 4.8 12.3 13c.2 3.7 0 7.42 0 12.18 6.3 0 11.92-.05 17.53 0 8.62.1 13.76 4.54 13.89 11.91.14 7.58-5.23 12.58-13.92 12.65-13.41.1-26.82 0-40.23 0-9.16 0-14.69 4.66-14.92 12.37-.24 8.09 5.15 13.11 14.58 13.34 8.87.22 17.78-.22 26.61.47 19.72 1.52 34.15 17.37 34.27 37a37.41 37.41 0 0 1-33.32 37.49Z"
              fill={
                {
                  "payload": 4278190080,
                  "type": 0,
                }
              }
            />
          </RNSVGGroup>
        </RNSVGSvgView>
        <Text
          style={
            {
              "fontFamily": "Tajawal Bold",
              "fontSize": 17,
              "fontWeight": "600",
              "marginLeft": 4,
            }
          }
        >
          -
        </Text>
        <RNSVGSvgView
          align="xMidYMid"
          bbHeight="14"
          bbWidth="14"
          fill="#6cdd6e"
          focusable={false}
          meetOrSlice={0}
          minX={0}
          minY={0}
          style={
            [
              {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              {
                "height": 14,
                "margin": 4,
                "width": 14,
              },
              {
                "flex": 0,
                "height": 14,
                "width": 14,
              },
            ]
          }
          vbHeight={260}
          vbWidth={260}
        >
          <RNSVGGroup
            fill={
              {
                "payload": 4285324654,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          >
            <RNSVGPath
              d="M197.48 113.71h-51.19V62.52a14.78 14.78 0 0 0-14.78-14.77h-3a14.78 14.78 0 0 0-14.78 14.77v51.19H62.52a14.78 14.78 0 0 0-14.77 14.78v3a14.78 14.78 0 0 0 14.77 14.78h51.19v51.19a14.78 14.78 0 0 0 14.78 14.77h3a14.78 14.78 0 0 0 14.78-14.77v-51.17h51.19a14.78 14.78 0 0 0 14.77-14.78v-3a14.78 14.78 0 0 0-14.75-14.8Z"
              fill={
                {
                  "payload": 4278190080,
                  "type": 0,
                }
              }
            />
          </RNSVGGroup>
        </RNSVGSvgView>
      </View>
      <View
        style={
          [
            {
              "alignItems": "center",
              "backgroundColor": "#2b2d33",
              "borderRadius": 50,
              "flexDirection": "row",
              "padding": 6,
            },
            {
              "borderRadius": 6,
              "justifyContent": "center",
              "marginHorizontal": 4,
              "paddingHorizontal": 8,
              "paddingVertical": 10,
            },
          ]
        }
      >
        <RNSVGSvgView
          align="xMidYMid"
          bbHeight="22"
          bbWidth="22"
          fill="#ede43b"
          focusable={false}
          meetOrSlice={0}
          minX={0}
          minY={0}
          style={
            [
              {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              [
                {
                  "height": 24,
                  "width": 24,
                },
                {
                  "height": 22,
                  "width": 22,
                },
              ],
              {
                "flex": 0,
                "height": 22,
                "width": 22,
              },
            ]
          }
          vbHeight={260}
          vbWidth={260}
        >
          <RNSVGGroup
            fill={
              {
                "payload": 4293780539,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          >
            <RNSVGPath
              d="M129.65 28.4c58.41-.29 103.08 46.14 102.19 100.83-1 63-50.36 102.54-102.37 102.37-58-.2-102-46.31-101.32-100.73C29 66.85 78.89 28.3 129.65 28.4ZM99.28 130a30.74 30.74 0 1 0 30.77-30.7A30.75 30.75 0 0 0 99.28 130Zm74.89-27.07L205.89 99c-9.35-21.48-24.35-36.4-45.89-45.41a128.86 128.86 0 0 0-2.15 32.25ZM54.16 161c9.31 21.51 24.31 36.45 45.91 45.42a131.11 131.11 0 0 0 2.13-32.23l-16.34-17.04ZM99 54.19c-21.44 9.37-36.21 24.42-45.16 45.89a126.07 126.07 0 0 0 32.28 2l16.77-16.2Q101.07 70.89 99 54.19Zm62.1 151.28c19.17-6.19 41.77-28.83 44.82-44.93a130.13 130.13 0 0 0-31.52-1.91l-17.07 16.26c1.21 9.83 2.47 20.11 3.77 30.58ZM123.3 79.08h13.85c.89-10.13 1.76-19.93 2.67-30.19A75.56 75.56 0 0 0 120 49c.25 10.5 1 20.43 3.3 30.08Zm88.05 40.8c-10.76.52-20.82 1-30.45 3.53v13.75l30.45 2.67Zm-162.07.38c-1.48 5.56-1.35 15.44.31 19.83a150.46 150.46 0 0 0 29.6-3.36v-13.55Zm71.27 91a73.42 73.42 0 0 0 19.53-.14c-.29-10.49-1.1-20.4-3.3-30h-13.72c-.85 10.19-1.66 19.95-2.51 30.1Z"
              fill={
                {
                  "payload": 4278190080,
                  "type": 0,
                }
              }
            />
          </RNSVGGroup>
        </RNSVGSvgView>
        <Text
          style={
            {
              "fontFamily": "Tajawal Bold",
              "fontSize": 17,
              "fontWeight": "600",
              "marginLeft": 4,
            }
          }
        >
          -
        </Text>
        <Text
          style={
            {
              "bottom": -6,
              "color": "#b3b7c9",
              "fontSize": 11,
              "fontWeight": "600",
              "letterSpacing": 2,
              "position": "absolute",
            }
          }
        >
          help
        </Text>
      </View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "backgroundColor": "#2b2d33",
            "borderRadius": 50,
            "flexDirection": "row",
            "opacity": 1,
            "padding": 6,
          }
        }
      >
        <RNSVGSvgView
          align="xMidYMid"
          bbHeight="24"
          bbWidth="24"
          fill="#ff5656"
          focusable={false}
          meetOrSlice={0}
          minX={0}
          minY={0}
          style={
            [
              {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              {
                "height": 24,
                "width": 24,
              },
              {
                "flex": 0,
                "height": 24,
                "width": 24,
              },
            ]
          }
          vbHeight={260}
          vbWidth={260}
        >
          <RNSVGGroup
            fill={
              {
                "payload": 4294923862,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          >
            <RNSVGPath
              d="M97.11 122.11a38 38 0 0 1 23.3-18.77c21.45-6.23 40.19 5.37 44.47 27.65 3.2 16.66.51 32.53-7.51 47.45-8.13 15.13-19 28.17-31.14 40.19-1.5 1.49-4.71 2.62-6.64 2.1-19.32-5.1-38.15-11.65-54.25-23.91-12.51-9.51-21.42-21.83-25-37.38-7.05-30.55 20.57-45.37 40.06-42.32 5.49.88 10.76 3.17 16.71 4.99Zm5.68-18.91a46 46 0 0 1 35.86-9c17.71 2.79 29.92 15.25 33.92 34.18 1.21 5.71 1.55 11.61 2.29 17.46a1.74 1.74 0 0 0 .72 0c18.63-8.34 33.06-21.07 41.3-40.07 3.48-8 5-16.45 3.08-25.13-3.6-16.51-21.7-28.74-39.14-26.09-6 .91-11.76 3.23-17.09 4.75-4.11-4.18-7.92-9.23-12.81-12.81-17.66-12.93-41.73-8.31-51.35 9.81-8.64 16.28-6.57 33.08-1.41 50.57 1.96-1.55 3.27-2.65 4.63-3.67ZM64.89 69a68.63 68.63 0 0 0 4.7 9.54 6.11 6.11 0 0 0 8.2 2.57 6.29 6.29 0 0 0 3.11-8.32 52.17 52.17 0 0 0-4.65-9.12 6.71 6.71 0 0 0-9.63-1.41A5.8 5.8 0 0 0 64.89 69ZM64 94.83a68.07 68.07 0 0 0-9.15 4.6 6.13 6.13 0 0 0-2.5 8.22 6.26 6.26 0 0 0 8.33 3 52.47 52.47 0 0 0 9.12-4.64 4.85 4.85 0 0 0 .61-.49 6.66 6.66 0 0 0 .89-9 6.11 6.11 0 0 0-7.3-1.69Zm120.52 85.42c-1.66-2.26-4.73-2.12-7.22-.52s-3.88 4.26-2.76 7.13a63 63 0 0 0 5.3 10.54c1.62 2.59 4.38 3.3 7.32 2.15h.08a6.63 6.63 0 0 0 3.17-9.32 84.51 84.51 0 0 0-5.86-9.98Zm18-20.87c2.22-1.63 1.95-4.79.4-7.25s-4.1-3.76-6.78-2.72a63.6 63.6 0 0 0-11.2 5.66 5.21 5.21 0 0 0-1.64 7.06 6.16 6.16 0 0 0 .63.91 7.72 7.72 0 0 0 9.61 1.72 72.29 72.29 0 0 0 8.99-5.38Z"
              fill={
                {
                  "payload": 4278190080,
                  "type": 0,
                }
              }
            />
          </RNSVGGroup>
        </RNSVGSvgView>
        <Text
          style={
            {
              "fontFamily": "Tajawal Bold",
              "fontSize": 17,
              "fontWeight": "600",
              "marginLeft": 4,
            }
          }
        >
          -
        </Text>
        <RNSVGSvgView
          align="xMidYMid"
          bbHeight="14"
          bbWidth="14"
          fill="#ff5656"
          focusable={false}
          meetOrSlice={0}
          minX={0}
          minY={0}
          style={
            [
              {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              {
                "height": 14,
                "margin": 4,
                "width": 14,
              },
              {
                "flex": 0,
                "height": 14,
                "width": 14,
              },
            ]
          }
          vbHeight={260}
          vbWidth={260}
        >
          <RNSVGGroup
            fill={
              {
                "payload": 4294923862,
                "type": 0,
              }
            }
            propList={
              [
                "fill",
              ]
            }
          >
            <RNSVGPath
              d="M197.48 113.71h-51.19V62.52a14.78 14.78 0 0 0-14.78-14.77h-3a14.78 14.78 0 0 0-14.78 14.77v51.19H62.52a14.78 14.78 0 0 0-14.77 14.78v3a14.78 14.78 0 0 0 14.77 14.78h51.19v51.19a14.78 14.78 0 0 0 14.78 14.77h3a14.78 14.78 0 0 0 14.78-14.77v-51.17h51.19a14.78 14.78 0 0 0 14.77-14.78v-3a14.78 14.78 0 0 0-14.75-14.8Z"
              fill={
                {
                  "payload": 4278190080,
                  "type": 0,
                }
              }
            />
          </RNSVGGroup>
        </RNSVGSvgView>
      </View>
    </View>
  </View>
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
        "marginVertical": 16,
      }
    }
  >
    <RNSVGSvgView
      align="xMidYMid"
      bbHeight="24"
      bbWidth="24"
      focusable={false}
      meetOrSlice={0}
      minX={0}
      minY={0}
      style={
        [
          {
            "backgroundColor": "transparent",
            "borderWidth": 0,
          },
          {
            "fill": "white",
            "height": 24,
            "width": 24,
          },
          {
            "flex": 0,
            "height": 24,
            "width": 24,
          },
        ]
      }
      vbHeight={260}
      vbWidth={260}
    >
      <RNSVGGroup
        fill={
          {
            "payload": 4294967295,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      >
        <RNSVGPath
          d="M130 38.3h63.49c17.26 0 28.3 11 28.32 28.27q.09 63.48 0 127c0 16.89-11.42 28.17-28.25 28.18h-127c-16.88 0-28.28-11.24-28.31-28.12q-.09-63.62 0-127.25c0-17 11.19-28 28.24-28q31.74-.11 63.51-.08Zm-72.13 91.46v62.89c0 7.38 1.87 9.29 9.14 9.29h126.11c7 0 9-2 9-9.13V67.03c0-7.18-1.92-9.15-9-9.16H66.71c-6.93 0-8.85 2-8.85 9Zm94.29-.21c11-6.48 16-17.9 13-29.18-3.31-12.07-12.91-19.37-25.87-19.52-8.28-.08-16.58-.26-24.84.17-9.06.46-15.62 5.11-19 13.61-1.56 3.9-2.12 8 1.12 11.52 2.91 3.15 6.72 4.69 10.51 2.66 3-1.62 5.29-4.66 9.19-8.28 6.83 0 15.59-.29 24.31.23 2 .12 4.55 2.69 5.52 4.76 1.29 2.75-.71 5.07-3.26 6.58q-8.12 4.79-16.23 9.58c-6.56 3.91-8.86 14-4.51 20.39a9.21 9.21 0 0 0 3.59 3c6.54 3 11.86-.16 14.45-8.73 4.04-2.28 8.08-4.46 12.02-6.79Zm-22.47 56A12.71 12.71 0 0 0 143 172.84c.11-7.88-5.09-13.31-12.87-13.41s-13 5.13-13.17 12.8 5.16 13.09 12.73 13.37Zm55.55 9.76a9.87 9.87 0 0 0 10.23-9.25c.43-5.23-4-10.07-9.46-10.35a9.81 9.81 0 0 0-.77 19.6Zm-101.15-9.8a9.8 9.8 0 1 0-9.66 9.87 10 10 0 0 0 9.66-9.87Zm91.82-111.18a10 10 0 0 0 9.9 9.63c5.23-.1 9.81-4.75 9.69-9.85a9.8 9.8 0 0 0-19.59.22Zm-111.41-.2a9.8 9.8 0 0 0 19.59.19 9.8 9.8 0 0 0-19.59-.19Z"
          fill={
            {
              "payload": 4278190080,
              "type": 0,
            }
          }
        />
      </RNSVGGroup>
    </RNSVGSvgView>
    <Text
      style={
        [
          {
            "fontFamily": "Tajawal ExtraBold",
          },
          {
            "fontSize": 20,
            "letterSpacing": 0.6,
            "marginHorizontal": 8,
          },
        ]
      }
    >
      Calisthenics Quiz
    </Text>
    <RNSVGSvgView
      align="xMidYMid"
      bbHeight="24"
      bbWidth="24"
      focusable={false}
      meetOrSlice={0}
      minX={0}
      minY={0}
      style={
        [
          {
            "backgroundColor": "transparent",
            "borderWidth": 0,
          },
          {
            "fill": "white",
            "height": 24,
            "width": 24,
          },
          {
            "flex": 0,
            "height": 24,
            "width": 24,
          },
        ]
      }
      vbHeight={260}
      vbWidth={260}
    >
      <RNSVGGroup
        fill={
          {
            "payload": 4294967295,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      >
        <RNSVGPath
          d="M130 38.3h63.49c17.26 0 28.3 11 28.32 28.27q.09 63.48 0 127c0 16.89-11.42 28.17-28.25 28.18h-127c-16.88 0-28.28-11.24-28.31-28.12q-.09-63.62 0-127.25c0-17 11.19-28 28.24-28q31.74-.11 63.51-.08Zm-72.13 91.46v62.89c0 7.38 1.87 9.29 9.14 9.29h126.11c7 0 9-2 9-9.13V67.03c0-7.18-1.92-9.15-9-9.16H66.71c-6.93 0-8.85 2-8.85 9Zm94.29-.21c11-6.48 16-17.9 13-29.18-3.31-12.07-12.91-19.37-25.87-19.52-8.28-.08-16.58-.26-24.84.17-9.06.46-15.62 5.11-19 13.61-1.56 3.9-2.12 8 1.12 11.52 2.91 3.15 6.72 4.69 10.51 2.66 3-1.62 5.29-4.66 9.19-8.28 6.83 0 15.59-.29 24.31.23 2 .12 4.55 2.69 5.52 4.76 1.29 2.75-.71 5.07-3.26 6.58q-8.12 4.79-16.23 9.58c-6.56 3.91-8.86 14-4.51 20.39a9.21 9.21 0 0 0 3.59 3c6.54 3 11.86-.16 14.45-8.73 4.04-2.28 8.08-4.46 12.02-6.79Zm-22.47 56A12.71 12.71 0 0 0 143 172.84c.11-7.88-5.09-13.31-12.87-13.41s-13 5.13-13.17 12.8 5.16 13.09 12.73 13.37Zm55.55 9.76a9.87 9.87 0 0 0 10.23-9.25c.43-5.23-4-10.07-9.46-10.35a9.81 9.81 0 0 0-.77 19.6Zm-101.15-9.8a9.8 9.8 0 1 0-9.66 9.87 10 10 0 0 0 9.66-9.87Zm91.82-111.18a10 10 0 0 0 9.9 9.63c5.23-.1 9.81-4.75 9.69-9.85a9.8 9.8 0 0 0-19.59.22Zm-111.41-.2a9.8 9.8 0 0 0 19.59.19 9.8 9.8 0 0 0-19.59-.19Z"
          fill={
            {
              "payload": 4278190080,
              "type": 0,
            }
          }
        />
      </RNSVGGroup>
    </RNSVGSvgView>
  </View>
  <View
    style={
      {
        "backgroundColor": "black",
        "borderBottomWidth": 2,
        "borderColor": "#484c56",
        "borderTopWidth": 2,
        "height": 200,
      }
    }
  >
    <Image
      source={
        {
          "testUri": "../../../src/assets/bootstrap-image.jpg",
        }
      }
      style={
        {
          "flex": 1,
          "height": undefined,
          "opacity": 0.4,
          "width": undefined,
        }
      }
    />
  </View>
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "padding": 12,
      }
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#1a1b1e",
          "borderRadius": 6,
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "center",
          "opacity": 1,
          "paddingHorizontal": 12,
          "paddingVertical": 16,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="20"
        bbWidth="20"
        fill="#e3e8ed"
        focusable={false}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "height": 20,
              "width": 20,
            },
            {
              "flex": 0,
              "height": 20,
              "width": 20,
            },
          ]
        }
        vbHeight={260}
        vbWidth={260}
      >
        <RNSVGGroup
          fill={
            {
              "payload": 4293126381,
              "type": 0,
            }
          }
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M41.76 116.79c-7.48-8.67-10.13-18.79-9.94-29.86.18-10.86 0-21.73 0-32.59 0-9 3.53-12.56 12.45-12.56h171.25c9 0 12.54 3.52 12.58 12.43 0 10.1-.21 20.2.06 30.29.32 12-1.83 23.14-10 32.29 0 24 .06 47.7 0 71.41-.05 15.08-9.73 26.74-24.38 29.53a34 34 0 0 1-6.38.47H72.65c-18.37 0-30.87-12.49-30.9-30.8-.02-23.4.01-46.81.01-70.61Zm48.88 81.55v-46c0-9.1 3.41-12.53 12.44-12.54h53.69c9.14 0 12.52 3.37 12.55 12.43v46.34h16c10 0 13.24-3.27 13.25-13.18v-56.3c-7.46 1.3-14.38 1.3-21.26-.77s-12.75-6.27-18-11.32c-8.15 8.55-17.78 12.95-29.23 13-11.62.07-21.26-4.69-29.46-13-10.8 11.14-23.73 14.93-39.16 12.11v56.65a28.29 28.29 0 0 0 .21 4.12c.74 4.76 3.59 8.07 8.36 8.36 6.62.41 13.33.1 20.61.1ZM51.78 61.65c0 10-.12 19.62 0 29.22.19 11.13 8.81 19.6 19.54 19.48A19.33 19.33 0 0 0 90.58 91c.17-8.86 0-17.73 0-26.6a19.85 19.85 0 0 0-.39-2.78Zm59.16.06a16.14 16.14 0 0 0-.36 2.37c0 9.18-.18 18.35 0 27.52a19.37 19.37 0 0 0 38.72.08c.14-3.36.06-6.73.06-10.09V61.71Zm58.79 0c0 11.31-.76 22.48.24 33.49.81 8.91 10.21 15.41 19.19 15.15 9.22-.27 18-7 18.59-15.92.76-10.76.18-21.62.18-32.72Zm-20.5 98h-38.32v38.51h38.32Z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <View
        style={
          {
            "width": 8,
          }
        }
      />
      <Text
        style={
          [
            {
              "fontFamily": "Tajawal Bold",
            },
            {
              "color": "#e3e8ed",
              "fontSize": 18,
              "letterSpacing": 2,
            },
          ]
        }
      >
        Store
      </Text>
    </View>
    <View
      style={
        {
          "width": 12,
        }
      }
    />
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#1a1b1e",
          "borderRadius": 6,
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "center",
          "opacity": 1,
          "paddingHorizontal": 12,
          "paddingVertical": 16,
        }
      }
    >
      <RNSVGSvgView
        align="xMidYMid"
        bbHeight="20"
        bbWidth="20"
        fill="#e3e8ed"
        focusable={false}
        meetOrSlice={0}
        minX={0}
        minY={0}
        style={
          [
            {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            {
              "height": 20,
              "width": 20,
            },
            {
              "flex": 0,
              "height": 20,
              "width": 20,
            },
          ]
        }
        vbHeight={260}
        vbWidth={260}
      >
        <RNSVGGroup
          fill={
            {
              "payload": 4293126381,
              "type": 0,
            }
          }
          propList={
            [
              "fill",
            ]
          }
        >
          <RNSVGPath
            d="M31.8 130c0-14 .05-28.07 0-42.11-.07-9.82 4.09-16.96 12.63-21.89q36.73-21 73.31-42.31C126 19 134 19 142.24 23.71Q179.11 45 216 66.28c8.23 4.73 12.22 11.83 12.21 21.28v85c0 9.46-4.07 16.51-12.3 21.24q-36.78 21.2-73.52 42.44c-8.3 4.8-16.42 4.82-24.74 0q-36.72-21.31-73.53-42.45c-8.24-4.73-12.33-11.74-12.33-21.21q.02-21.32.01-42.58Zm22.34.06v41a4 4 0 0 0 2.31 4.07q35.69 20.48 71.26 41.14a4 4 0 0 0 4.7 0q35.7-20.74 71.48-41.29a3.42 3.42 0 0 0 2.06-3.43q-.09-41.56 0-83.15a3.25 3.25 0 0 0-1.83-3.29q-36-20.7-71.95-41.51a3.4 3.4 0 0 0-4-.11Q92.08 64.42 56 85.16a3.22 3.22 0 0 0-1.84 3.28c.01 13.86-.01 27.72-.02 41.56Zm75.91-44.6c24.48-.17 44.43 19.79 44.55 44.57a44.6 44.6 0 1 1-44.55-44.57Zm22.3 44.67A22.35 22.35 0 1 0 130 152.35a22.49 22.49 0 0 0 22.35-22.26Z"
            fill={
              {
                "payload": 4278190080,
                "type": 0,
              }
            }
          />
        </RNSVGGroup>
      </RNSVGSvgView>
      <View
        style={
          {
            "width": 8,
          }
        }
      />
      <Text
        style={
          [
            {
              "fontFamily": "Tajawal Bold",
            },
            {
              "color": "#e3e8ed",
              "fontSize": 18,
              "letterSpacing": 2,
            },
          ]
        }
      >
        Settings
      </Text>
    </View>
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#1a1b1e",
        "borderRadius": 6,
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
        "marginBottom": 12,
        "marginHorizontal": 12,
        "opacity": 1,
        "paddingHorizontal": 12,
        "paddingVertical": 16,
      }
    }
  >
    <RNSVGSvgView
      align="xMidYMid"
      bbHeight="20"
      bbWidth="20"
      fill="#4fbeff"
      focusable={false}
      meetOrSlice={0}
      minX={0}
      minY={0}
      style={
        [
          {
            "backgroundColor": "transparent",
            "borderWidth": 0,
          },
          {
            "height": 20,
            "width": 20,
          },
          {
            "flex": 0,
            "height": 20,
            "width": 20,
          },
        ]
      }
      vbHeight={260}
      vbWidth={260}
    >
      <RNSVGGroup
        fill={
          {
            "payload": 4283416319,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      >
        <RNSVGPath
          d="M68.68 225H62c-9.67-3.39-13-10.91-13.41-20.41-.06-1.61-.14-3.22-.14-4.82V60.27a76.27 76.27 0 0 1 .35-8.15c1.55-14.22 12.9-20.78 26-15a69.25 69.25 0 0 1 7 3.45q61.11 35 122.17 70.06a74.34 74.34 0 0 1 7.45 4.86c9.88 7.36 11 17.89 2.46 26.68a43.22 43.22 0 0 1-9.49 7q-56.84 32.76-113.81 65.26C83.5 218.37 76 221.48 68.68 225Z"
          fill={
            {
              "payload": 4278190080,
              "type": 0,
            }
          }
        />
      </RNSVGGroup>
    </RNSVGSvgView>
    <View
      style={
        {
          "width": 12,
        }
      }
    />
    <Text
      style={
        [
          {
            "fontFamily": "Tajawal ExtraBold",
          },
          {
            "color": "#4fbeff",
            "fontSize": 22,
          },
        ]
      }
    >
      Tap to start
    </Text>
  </View>
</View>
`;
